commerce-test/src/testIntegration/java/com/liferay/commerce/discount/test/CommerceDiscountTest.java

```diff
@@ -79,6 +88,21 @@ public void setUp() throws Exception {
 		_commerceAccount =
 			_commerceAccountLocalService.getPersonalCommerceAccount(
 				_user.getUserId());
+
+		_commerceOrders = new ArrayList<>();
+	}
+
+	@After
+	public void tearDown() throws Exception {
+		for (CommerceOrder commerceOrder : _commerceOrders) {
+			CommerceOrderLocalServiceUtil.deleteCommerceOrder(commerceOrder);
+		}
+
+		CommerceDiscountLocalServiceUtil.deleteCommerceDiscounts(
+			_group.getCompanyId());
+		CommerceAccountLocalServiceUtil.deleteCommerceAccount(_commerceAccount);
+		GroupLocalServiceUtil.deleteGroup(_group);
+		UserLocalServiceUtil.deleteUser(_user);
```

Comment:
It'd be better to inject the `*LocalService` references here, rather than calling the `*LocalServiceUtil`s, especially for the sake of consistency with the rest of this file.